doozer/doozerlib/constants.py

```diff
@@ -29,6 +29,12 @@
 KONFLUX_REPO_CA_BUNDLE_TMP_PATH = "/tmp"
 KONFLUX_REPO_CA_BUNDLE_FILENAME = "Current-IT-Root-CAs.pem"
 KONFLUX_REPO_CA_BUNDLE_HOST = "https://certs.corp.redhat.com/certs"
+WORKING_SUBDIR_KONFLUX_BUILD_SOURCES = "konflux_build_sources"
```

Comment:
Noticed that these constants aren't being used? Or not sure if I missed something or it'll be in an another PR.

Reply:
right, it is used in https://github.com/openshift-eng/art-tools/pull/820